cli/src/cmd/forge/script/runner.rs

```diff
@@ -72,7 +72,11 @@ impl ScriptRunner {
             traces: constructor_traces,
             debug: constructor_debug,
             ..
-        } = self.executor.deploy(CALLER, code.0, 0u32.into(), None).expect("couldn't deploy");
+        } = self
+            .executor
+            .deploy(CALLER, code.0, 0u32.into(), None)
+            .map_err(|err| eyre::eyre!("Constructor failed:\n{}", err))?;
```

Comment:
"Failed to deploy contract"?

Reply:
should allude to the constructor coming from the script contract

Reply:
yeh good idea to add info that this is a script issue